### PR TITLE
Add glossaries working group

### PR DESCRIPTION
**Pull request by barbaricyawps**
## Purpose / why

Add the glossaries working group to the working group page.

## What changes were made?

New working group page for glossaries.

### CODE REVIEW

**Review by bwklein — APPROVED** [review]
Looks good to me.
https://deploy-preview-58--gdp-hugo.netlify.app/working-group/glossaries/

File: content/en/working-group/glossaries.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+title: Glossaries
+poc: "[Cameron Shorter](https://thegooddocs.slack.com/team/UKTGLQNGG)"
+meeting: Monthly on Tuesdays at 7AM UTC(on Google Meet)
+status: Active
+description: Creating cross-organizational and inter-operable glossaries with a standardised format for terms and definitions.
+slackName: glossaries
+slackID: C019A1EQJMC
+notesURL: https://docs.google.com/document/d/1Fjrl34ErnYammel9WmvXJ3rMWFANjoSiiGyyNSYOXUg/
```
Comment (camerons) [inline content/en/working-group/glossaries.md:9]:
```suggestion
notesURL: https://docs.google.com/document/d/1yzncnQB7uYktPnAQ4DqZYVPDdYnqDrWmqHzcpXrjkPg/
```
I think this is supposed to reference our meeting notes page, as per above.

File: content/en/working-group/glossaries.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  - "[Codrina Maria Ilie](https://thegooddocs.slack.com/team/D01A8SW6CBX)"
+  - "[Reese W. Plews](https://thegooddocs.slack.com/archives/D019RSVPE69)"
+  - "[Rob Atkinson](https://thegooddocs.slack.com/team/U01DYRWG43X)"
+
+draft: false
+---
+
+The goal of our community is to create a template for glossaries that can be easily integrated by anyone creating glossaries for their own project. The pilot will build processes and tools for the generic use case, developing and applying them within the geospatial mapping domain.
+
+We plan to use the base template (a "template of templates") that we've created as the starting point to create a template for glossaries. We will also create a process for a central glossary owner to accept, triage, and refine proposals for new terms from their community.
```
Comment (camerons) [inline content/en/working-group/glossaries.md:22]:
```suggestion
* We plan to use the base template (a "template of templates") that we've created as the starting point to create a template for glossaries.
* We will establish processes for sharing terms between organizations and glossaries.
* We will create a process for a central glossary owner to accept, triage, and refine proposals for new terms from their community.
```
Adding in the use case of sharing terms between organizations.

File: content/en/working-group/glossaries.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
+---
+title: Glossaries
+poc: "[Cameron Shorter](https://thegooddocs.slack.com/team/UKTGLQNGG)"
+meeting: Monthly on Tuesdays at 7AM UTC(on Google Meet)
```
Comment (camerons) [inline content/en/working-group/glossaries.md:4]:
```suggestion
meeting: Monthly on third Tuesdays of the month, at 7AM UTC (on Google Meet)
```

**Review by camerons — CHANGES_REQUESTED** [review]
A few suggested changes.

**Review by techtrailhead — APPROVED** [review]

File: content/en/working-group/glossaries.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+  - "[Ankita Tripathi](https://thegooddocs.slack.com/team/D016LHJPDSR)"
+  - "[Cameron Shorter](https://thegooddocs.slack.com/team/UKTGLQNGG)"
+  - "[Codrina Maria Ilie](https://thegooddocs.slack.com/team/D01A8SW6CBX)"
+  - "[Reese W. Plews](https://thegooddocs.slack.com/archives/D019RSVPE69)"
+  - "[Rob Atkinson](https://thegooddocs.slack.com/team/U01DYRWG43X)"
+
+draft: false
+---
+
+The goal of our community is to create a template for glossaries that can be easily integrated by anyone creating glossaries for their own project. The pilot will build processes and tools for the generic use case, developing and applying them within the geospatial mapping domain.
```
Comment (bwklein) [inline content/en/working-group/glossaries.md:20]:
```suggestion
The goal of our community is to create a template for glossaries that can be easily integrated by anyone creating glossaries for their own project.
The pilot will build processes and tools for the generic use case, developing and applying them within the geospatial mapping domain.
```

**Review by bwklein — APPROVED** [review]

**Comment by bwklein** [comment]
@barbaricyawps It seems that all of @camerons changes have been incorporated.  I have re-requested a review from him to be sure.  Otherwise, it looks good to go.

**Comment by barbaricyawps** [comment]
Merging this in!

**Pull request merged**